### DISCUSSION TITLE
Mark all as read when scrolled to bottom

### DISCUSSION THREAD

**Discussion by UncleArya** [jocmp/capyreader#608] [Feature Requests]
I absolutely love the recently added feature that auto marks articles as read when you scroll past them, that is one of the best QoL features for me personally. Thank you for adding that! One suggestion to further improve it would be when you get to the bottom of the reading list, everything will have been marked as read except the last 2-3 furthest to the bottom that don't have a chance to get off screen. So possibly an option to have all articles get auto marked as read on reaching the bottom or adding the ability to scroll past them to a blank screen so they can get marked too?

**Comment by jocmp** [comment]
hey @UncleArya I'm glad it's been working for you! Good catch with the end-of-feed behavior. I added a card to fix this here https://github.com/jocmp/capyreader/issues/609

**Comment by jocmp** [comment]
This is now available in 2024.12.1084-dev.

**Comment by Offerel** [comment]
Sorry for capturing this discussion. But i have a additional request regarding the same context. Would it be possible to add a "Mark all as read" button after the last article in the list? I know there is a similar button in the top bar with exact this functionality. But in my opinion, it would be nice to have a copy of this button at the bottom of the list, after all articles.
